### ISSUE TITLE
Joining Campaign issue

### ISSUE THREAD

**Issue by threeheadedinternet** [worldanvil/worldanvil-bug-tracker#452]
**World Anvil Username**:
threeheadedinternet

**World / Campaign / RPG Character / Manuscript Title**:
Night City Cyberpunk RED Campaign

**Feature**: 
Joining Campaign

**Related URLS** (To public & visible pages add screenshots otherwise):
https://www.worldanvil.com/epic/zDCCTPdp4UrzSEiASWuEu0OIQ3WUUO

**Describe the Issue**
User ant_aghanim have two characters in the campaign, however one character somehow ended in campaign limbo. The character was not added to the campaign, but did get added to the campaign party. Because he is in the party, re-joining campaign produces 404. Trying to edit party, to kick the character, the character doesn't show up. Meaning we have a character that only shows up "cosmetically" on the party screen, but isn't in the party or campaign as far as functionality goes.

**To Reproduce**
1. Have not been able to reproduce on test account, account of world-owner, or world co-author
2. User and character was created, then joined world from link
3. User was then added as 'subscriber' to world to allow access to private articles
4. Somewhere in between this and second character joining world the first character gets bugged 

**Expected behavior**
The character should have joined the campaign, getting access to both campaign and party screens, as well as showing up on both campaign and party screen. We have fixed it by deleting the character. Second attempt worked as expected.

**Screenshots**
If applicable, add screenshots to help explain your problem. You can add screenshots by drag & drop into this window!

**Comment by iamromeo** [comment]
Hey @threeheadedinternet just to clarify - removing the character from the Party Settings tab does not work?

**Comment by threeheadedinternet** [comment]
@iamromeo - yeah it was a weird one, the character showed up on the party overview both for other players and world authors when looking at the campaign tabs, but not in any of the party settings on the GM side of things, meaning I didn't even have the option to remove him. I tried to tag the character in manually and got nothing. He also couldn't leave because nothing campaign-related showed up on his end.

We are two players and two authors in our group, and this is what has me stumped on it. Both normal players joined at the same time, added their characters at the same time, and were set as world subscribers by me at the same time. But this only affected one of them, and none of us have been able to reproduce it.

He sadly did delete the character (I would have told him to leave it for you guys to look at, but he beat me to it) but this is the second character that was added by that player after about a week of first joining, at which point I noticed the bug: https://www.worldanvil.com/hero/5a842507-dba5-42b6-856d-b31a591fa9b7

His remade character, which has joined up completely as expected, is here:
https://www.worldanvil.com/hero/06e781ee-152d-4c67-ae0e-92d9a2a857d4
